theano/configdefaults.py

```diff
@@ -242,6 +242,14 @@ def filter_nvcc_flags(s):
              FloatParam(0),
              in_c_key=False)
 
+AddConfigVar('gpuarray.single_stream',
+             """Switch between single stream mode or multi-stream
+             mode.  If your computation can't benefit from multiple
+             streams, single-stream is usually faster (by about 10%)
```

Comment:
Where this 10% come from? My understanding was that the old back-end was speed 1x. WIth single stream, it is 0.9x. With multiple stream, it was > 1x, like 1.1x. So the new back-end multi-stream vs single-stream is more then 10?.